src/NzbDrone.Core/Indexers/IndexerStatus.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NzbDrone.Core.Datastore;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.Indexers
+{
+    public class IndexerStatus : ModelBase
+    {
+        public int IndexerId { get; set; }
+
+        public DateTime? FirstFailure { get; set; }
+        public DateTime? LastFailure { get; set; }
```

Comment:
FirstFailure = first failure in the current escalation cycle..
LastFailure = last failure in the current escalation cycle.

I can change LastFailure to MostRecentFailure, but then what should FirstFailure be renamed to?